### DISCUSSION TITLE
Scene Zero

### DISCUSSION THREAD

**Discussion by emandtf** [Koenkk/zigbee2mqtt#23989] [General]
I've bought Aqara Cube and discovered that sometimes it seems triggering Scene Zero which turn_off all devices that have OnOff cluster.
Is there a way to disallow that behavior?
(I'm not using Scenes at all)

**Comment by Koenkk** [comment]
Could you try to send to `zigbee2mqtt/FRIENDLY_NAME/set` payload `{"scene_remove": 0}`? Does this fix the issue? Make sure to wakeup the device right before sending the command, otherwise you will get a timeout error.

**Reply by emandtf** [reply]
Sent this to what device? The Cube or ALL other devices?

I'm asking this because I never used, enabled and/or set/configured any Scene so I don't understand why all other devices have to react to a command that is not shown and neither was set.....
